### ISSUE TITLE
Remove `napari-btrack-reader` from the hub

### ISSUE THREAD

**Issue by paddyroddy** [chanzuckerberg/napari-hub#920]
The functionality of [napari-btrack-reader](https://www.napari-hub.org/plugins/napari-btrack-reader) was merged into another repo quantumjot/BayesianTracker#115, which exists now in [btrack](https://www.napari-hub.org/plugins/btrack). We are now deprecating this package and this is one of the steps we need lowe-lab-ucl/napari-btrack-reader#4

**Comment by dgmccart** [comment]
Hi Patrick, you can remove your plugin from the hub by specifying the visibility as hidden in the plugin manifest. [Here is documentation](https://github.com/chanzuckerberg/napari-hub/wiki/Customizing-your-plugin's-listing#visibility) with more info.

**Comment by richaagarwal** [comment]
@paddyroddy Actually, I see you've already archived that github repo, in which case we can just go ahead and manually "block" the plugin.  I'll report back once that change is complete on our end.

**Comment by paddyroddy** [comment]
I received this in my email:

> [@paddyroddy](https://github.com/paddyroddy) Actually, I see you've already archived that github repo, in which case we can just go ahead and manually "block" the plugin. I'll report back once that change is complete on our end.

I've just searched in the napari hub and can't find it, so this is done I assume?

**Comment by richaagarwal** [comment]
@paddyroddy Yes, we can close this out!  Apologies for the confusion on my last (deleted) comment - it turns out we didn't need to do anything on our end, as the work you all did to remove the napari trove classifier with its last release actually removed it from the hub.

**Comment by paddyroddy** [comment]
> @paddyroddy Yes, we can close this out! Apologies for the confusion on my last (deleted) comment - it turns out we didn't need to do anything on our end, as the work you all did to remove the napari trove classifier with its last release actually removed it from the hub.

Ah, okay. Glad it's all sorted!
